variables.tf

```
@@ -176,8 +254,24 @@ variable "security_group_rules" {
 
 variable "access_points" {
   description = "A map of access point definitions to create"
-  type        = any
-  default     = {}
+  type = map(object({
+    name = optional(string)
+    tags = optional(map(string))
```

since we typical merge tags with other tags we'll always need a default empty map for any tags variables/arguments:

```suggestion
    tags = optional(map(string), {})
```